test/quantization/test_numeric_suite_fx.py

```diff
@@ -285,3 +382,32 @@ def compare_and_validate_results(float_model, q_model, data):
 
         linear_data = self.calib_data[0][0]
         compare_and_validate_results(backup_prepared_model, q_model, linear_data)
+
+    @override_qengines
+    def test_compare_model_outputs_linear_dynamic_fx(self):
+        r"""Compare the output of linear layer in dynamic quantized model and corresponding
+        output of linear layer in float model
+        """
+
+        def compare_and_validate_results(float_model, q_model, data):
+            act_compare_dict = compare_model_outputs_fx(float_model, q_model, data)
+            print(act_compare_dict)
+            expected_act_compare_dict_keys = {"stats"}
```

Comment:
Expected key should have another one for fc1.

Reply:
Fixed.